src/layers/src/geojson-layer/geojson-layer.ts

```diff
@@ -429,11 +504,15 @@ export default class GeoJsonLayer extends Layer {
     const pickable = interactionConfig.tooltip.enabled;
     const hoveredObject = this.hasHoveredObject(objectHovered);
 
-    return [
-      new DeckGLGeoJsonLayer({
+    const {data, ...props} = dataProps;
+    const deckLayerData = this.isArrow ? data : [data];
```

Comment:
- Why is non-arrow data now wrapped in an array?
- The goal must be to remove the use of `isArrow`, presumably rely on container as much as possible?

Reply:
It's because arrow table can have multiple chunks/batches, and a deck.gl layer is created for each chunk.